Failed Helm release caused by unresolved dependency

## Describe the bug

The Helm releaser fails during the packaging process.

### Expected behavior

The Helm charts should be released successfully without errors.

### Actual behavior

The chart releaser fails when attempting to package the charts/ssi-dim-wallet-stub chart due to a missing repository definition.

### Screenshots/Error Messages

```bash
Run helm/chart-releaser-action@v1.4.1
Run owner=$(cut -d '/' -f 1 <<< "$GITHUB_REPOSITORY")
Looking up latest tag...
Discovering changed charts since '0.0.6'...
Installing chart-releaser on /opt/hostedtoolcache/cr/v1.4.1/x86_64...
Adding cr directory to PATH...
Packaging chart 'charts/ssi-dim-wallet-stub-memory'...
Successfully packaged chart in /home/runner/work/ssi-dim-wallet-stub/ssi-dim-wallet-stub/charts/ssi-dim-wallet-stub-memory and saved it to: .cr-release-packages/ssi-dim-wallet-stub-memory-0.1.10.tgz
Packaging chart 'charts/ssi-dim-wallet-stub'...
Error: no repository definition for https://charts.bitnami.com/bitnami
Usage:
  cr package [CHART_PATH] [...] [flags]

Flags:
  -h, --help                     help for package
      --key string               Name of the key to use when signing
      --keyring string           Location of a public keyring (default "~/.gnupg/pubring.gpg")
  -p, --package-path string      Path to directory with chart packages (default ".cr-release-packages")
      --passphrase-file string   Location of a file which contains the passphrase for the signing key. Use '-' in order to read from stdin
      --sign                     Use a PGP private key to sign this package

Global Flags:
      --config string   Config file (default is $HOME/.cr.yaml)

Error: Process completed with exit code 1.
```

--- a/.github/workflows/helm-release.yml
+++ b/.github/workflows/helm-release.yml
@@ -51,6 +51,7 @@ jobs:
       - name: Update helm SSI-DIM-WALLET-STUB-DATABASE dependencies
         run: |
           cd charts/ssi-dim-wallet-stub
+          helm repo add bitnami https://charts.bitnami.com/bitnami
           helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
           helm dependency update
 

